.kube-linter.yaml

```diff
@@ -0,0 +1,10 @@
+
+checks:
+  addAllBuiltIn: true
+  exclude:
```

Comment:
Could we a few brief notes indicating why we've opted to exclude these?

I think `unset-*-requirements` is currently due to a bug that Rafal is actively fixing. The others I'm not super familiar with. If we're just disabling them to get passing results can we file issues for each ignored rule to eventual try and fix?

Reply:
Sure, i mean i want this to be something we all participate on so feel free to add in another PR things you wish to expand on etc... I'll add my own thoughts but does not mean we have to keep those in if we do not want.